### PR TITLE
Add suggestion to also target .NET Standard 2.0

### PR DESCRIPTION
**Pull request by LorenzCK**
Short addition to the .NET Standard targeting guide.
Based on [.NET Standard targeting suggestions](https://github.com/dotnet/standard/blob/master/docs/versions.md#how-do-i-know-which-net-standard-version-i-should-target) on dotnet/standard and [tweet by David Fowler](https://twitter.com/davidfowl/status/1008453902058917888).

### CODE REVIEW

**Review by terrajobst — APPROVED** [review]
LGTM. We should probably update the link to point to the guidance introduced by #7833.

File: docs/standard/net-standard.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ In general, we recommend you to target the *lowest* version of .NET Standard pos
 1. Target the next lower version of .NET Standard and build your project.
 2. If your project builds successfully, repeat step 1. Otherwise, retarget to the next higher version and that's the version you should use.
 
+However, since targeting lower .NET Standard versions introduces a number of support dependencies, if your project targets .NET Standard 1.x, it is [suggested to *also* target .NET Standard 2.0](https://github.com/dotnet/standard/blob/master/docs/versions.md#how-do-i-know-which-net-standard-version-i-should-target). This will simplify the dependency graph for users of your library that run on .NET Standard 2.0 compatible frameworks and it will reduce the number of packages they will need to download.
```
Comment (rpetrusha) [inline docs/standard/net-standard.md]: it is suggested to also target --> we recommend that you also target 
will simplify --> simplifies
comma after "compatible frameworks"
will reduce --> reduces
will need --> need

**Review by rpetrusha — DISMISSED** [review]
Thanks for adding this, @LorenzCK. I've suggested a number of grammatical changes. 

@terrajobst had also suggested adding a link to the document in #7833, If you want to do that, the url will be ~/docs/standard/library-guidelines/breaking-changes.md. In that case, we'll merge your PR after #7833 merges.

**Comment by terrajobst** [comment]
@LorenzCK 

>  I pointed the link to the [`library-guidance/cross-platform-targeting.md` page](https://github.com/dotnet/docs/pull/7833/files#diff-86f904078e0352133c58e3f941af80f2) in #7833, which appears to be the correct document on how to target .NET Standard.

Yep, that's the document I had in mind too.

**Comment by LorenzCK** [comment]
Thanks @terrajobst, fixed.

**Review by terrajobst — APPROVED** [review]
LGTM now. @rpetrusha?

**Review by rpetrusha — APPROVED** [review]
This looks good, @LorenzCK. Thanks for both submitting the PR and making the changes. We'll merge now. Your changes will be live on docs.microsoft.com in the next day or two.

**Comment by terrajobst** [comment]
❤️

**Comment by LorenzCK** [comment]
Great! Thank you all 😊

**Pull request merged**